forc-plugins/forc-node/src/consts.rs

```diff
@@ -0,0 +1,17 @@
+/// Minimum fuel-core version supported.
+pub const MIN_FUEL_CORE_VERSION: &str = "0.40.0";
+
+pub const TESTNET_RESERVED_NODE: &str = "/dns4/p2p-testnet.fuel.network/tcp/30333/p2p/16Uiu2HAmDxoChB7AheKNvCVpD4PHJwuDGn8rifMBEHmEynGHvHrf";
```

Comment:
We should use the same setup for testnet that we do for mainnet. Primarily - testnet should use bootstrap nodes the same as mainnet does.

It should also use `/dnsaddr/testnet.fuel.network.` instead of a specific node like above.

Reply:
Sounds good to me! Fixing it now, we might want to update the this docs as well as I was following this one: https://docs.fuel.network/guides/running-a-node/running-a-testnet-node/